@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -52,7 +52,6 @@ class CfgVehicles
 		driverleftleganimname = "pedalL";
 		driverrighthandanimname = "p_cyclic";
 		driverrightleganimname = "pedalR";
-		simulation=helicopterRTD;
```

Comment:
This one still needs to be there, though you were correct in removing the other. By removing the other, that class will inherit its simulation from this one.